### DISCUSSION TITLE
<Copilot bugs in Rider IDE>

### DISCUSSION THREAD

**Discussion by DKurepin** [community/community#45145] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello! Recently, I've been working in Rider with GitHub Copilot and I've noticed some bugs and issues. Fisrtly, when you write a code sometimes Copilot just switches off randomly and then doesn't work at all. Also, when you write something it can log out from GitHub or smth. I would like you to report those issues, maybe it will help you in the future to fix it. Thanks for your attention and have a great day!

**Comment by iamcjwilliams** [comment]
@DKurepin Thank you for your feedback, Can you provide more details to be able to replicate the errors? This will help the copilot team pinpoint the issue you're experiencing

**Reply by DKurepin** [reply]
Hello! Thank you for the reply. Basically, this issues occur when I work on my project in c# file in JetBrains Rider. The problem with switching off happens when you writing a code (so the icon of the Copilot in the bottom of screen is becoming grey and when you pointing on that icon it says that Copilot is switched off. The second problem have never occurred since the newest update, but the issues was basically about accidental and autonomous logging out. I don't have the idea what is the cause of this problem, but maybe it's just IDE

**Reply by iamcjwilliams** [reply]
@DKurepin Are you still experiencing this issue?

**Reply by DKurepin** [reply]
> @DKurepin Are you still experiencing this issue?

Yeah, not so often, but sometimes it happens

**Comment by rlalance** [comment]
I'm having the same issue here.
New subscription, but I noticed that, I'm getting logged out constantly after login in.

**Reply by DKurepin** [reply]
Hi! Sometimes, I still have some troubles like you told with pycharm or rider, so maybe it's a common problem with the current/previous copilot updates

**Comment by rlalance** [comment]
I downloaded vscode, authenticated and completion works on the same C# file.
So... looks like a rider plugin issue.

**Reply by DKurepin** [reply]
I suppose so

**Reply by rlalance** [reply]
Uninstalled the plugin, re-installed the plugin, now it is staying logged in.
Umm.

**Reply by DKurepin** [reply]
Thanks for the advice, I will try!

**Reply by rlalance** [reply]
Good luck! Love copilot!!! :D

**Reply by DKurepin** [reply]
Yeah:) I am using copilot for a month now and it helps a lot
